apps/meteor/server/lib/rooms/roomTypes/livechat.ts

```diff
@@ -28,7 +27,7 @@ roomCoordinator.add(LivechatRoomType, {
 	},
 
 	canAccessUploadedFile({ rc_token: token, rc_rid: rid }) {
-		return token && rid && LivechatRooms.findOneOpenByRoomIdAndVisitorToken(rid, token);
+		return token && rid && !!Promise.await(LivechatRooms.findOneOpenByRoomIdAndVisitorToken(rid, token));
```

Comment:
is this used on client as well?

Reply:
It's used on FileUpload, the forbidden class